doc/development/deprecations.rst

```diff
@@ -131,6 +131,12 @@ Other deprecations
 Completed deprecation cycles
 ----------------------------
 
+* The ``qml.workflow.set_shots`` helper func has been removed. We no longer interact with the legacy device interface in our code 
+workflow behavior. Instead, shots should be specified on the tape, and the device should pull it's number of shots from the tape.
```

Comment:
The indentation is needed for the sphinx build to succeed.

```suggestion
  workflow behavior. Instead, shots should be specified on the tape, and the device should pull it's number of shots from the tape.
```

Reply:
I don't see a corresponding removal for the same deprecation from the "Pending deprecations" section. Though I can't find it either. Did we forget to add this to the deprecations page when it was first deprecated? 🤔

Reply:
I think it was never added: https://github.com/PennyLaneAI/pennylane/pull/6250/

Reply:
Agree. I couldn't find it anywhere. According to timestamps I deduced it to be 0.38 but now I think I was wrong and it should be 39.